policy/modules/apps/cdrecord.te

```diff
@@ -15,7 +15,20 @@ policy_module(cdrecord, 2.6.1)
 ## </desc>
 gen_tunable(cdrecord_read_content, false)
 
+## <desc>
+##	<p>
+##	Determine whether the cdripper
+##	programs can read various content,
+##	including user home directories,
+##	user temporary directories, nfs,
+##	samba, devices and untrusted
+##	content files
+##	</p>
+## </desc>
+gen_tunable(cdripper_rw_content, false)
+
```

Comment:
I suppose this is debatable. I think this is overdoing it. I would personally probably just allow it to atleast manage generic home content and maybe also generic tmp content unconditionally and get it over with.

Reply:
This reflects the current cdrecord role / local policy.

If cdrecord_t does not currently allow to even *read* home content, why should cdripper_t do so ?

This is another nack.

Reply:
cdrecord is allowed to read generic user home content files unconditionally, because that is what it does. it enables you to write a home content file to a cd.

cdripper should be allowed to manage generic user home content files unconditionally, because that is what it does. it enables you to rip a cd to a home content file.

Reply:
This is a NACK again.

cdrecord is NOT allowed to read generic user home content unconditionally, but only when the **cdrecord_read_content** is turned **ON** !

The equivalent (not the same) applies to the CD ripper program introduced in this patch:

tunable_policy(`cdripper_rw_content',`
        userdom_manage_user_home_content_files(cdripper_t)
        userdom_manage_user_home_content_symlinks(cdripper_t)
',`
        userdom_dontaudit_list_user_home_dirs(cdripper_t)
        userdom_dontaudit_manage_user_home_content_files(cdripper_t)
')

Reply:
Have a look:

https://github.com/SELinuxProject/refpolicy/blob/master/policy/modules/apps/cdrecord.te#L58